### PR TITLE
Update CLI to a) use v2 and b) do fixes described in redesign

### PR DESCRIPTION
**Pull request by lgnashold**
https://docs.google.com/document/d/1N6CR9s3UiDHF3VT5zocKB0Gct3brDfu-cDP4C8pZ8Y0/

### CODE REVIEW

File: vals/cli/suite.py [hunk 1]
```diff
@@ -18,151 +20,134 @@ def suite_group():
...
+
+    for i, suite in enumerate(suites):
+        truncated_title = (
+            suite.title[: title_width - 3] + "..."
+            if len(suite.title) > title_width
+            else suite.title
+        )
+        date_str = suite.last_modified_at.strftime("%Y/%m/%d %H:%M")
+        row = f"| {i + 1:{number_width}} | {truncated_title:{title_width}} | {suite.id:{id_width}} | {date_str:{last_modified_width}} |"
+        click.echo(row)
```
Comment (alexpattison) [inline vals/cli/suite.py:101]: have you tried using any command line formatting utility libraries? seems like this is getting a little complex

Comment (alexpattison) [reply vals/cli/suite.py:101]: lol actually looks like `click` is exactly that... just seems surprisingly verbose

**Review by alexpattison — APPROVED** [review]
LGTM

File: vals/cli/suite.py (hunk 1 above)
Comment (lgnashold) [reply vals/cli/suite.py:101]: I don't think click supports this. 
https://kdeldycke.github.io/click-extra/tabulate.html There is this but it's much less maintained. I think probably not adding a whole extra depedency just for this.

**Pull request merged**